edsdme/blocks/logos/LogosCardsStyles.js

```diff
@@ -0,0 +1,33 @@
+import { getLibs } from '../../scripts/utils.js';
+
+const miloLibs = getLibs();
+const { css } = await import(`${miloLibs}/deps/lit-all.min.js`);
+
+const borderColor = css`#eaeaea`;
+
+// eslint-disable-next-line import/prefer-default-export
+export const logosCardsStyles = css`
+
+  .partner-cards-collection {
```

Comment:
@cod17828 I'm surprised to see these generic css styles in the logo block.
I think if it also applies to other cardscollections, it should be added to the card collection css.
If it's specific for logos, we should add a class to indicate that this is for logos.

Reply:
since there is nothing specific for logos than it is for search card i removed the LogosCardsStyles.js